src/ToDo.UI/lists.json

```diff
@@ -23,4 +23,4 @@
     "wellknownListName": "none",
     "id": "List-Groceries"
   }
-]
+]
```

Comment:
Why this change? Can be removed from your PR no?

Reply:
yeap